packages/core/lib/src/core_widget_factory.dart

```diff
@@ -639,13 +639,15 @@ class WidgetFactory {
         _tagA ??= TagA(this).buildOp;
         meta.register(_tagA!);
```

Comment:
Maybe we should check for `href` before registering the build op?

Reply:
I didn't know, i didn't spend enough time on the code to suggest a better solution :/